docs/app/pages/components/components-sections/tables/reorderable-table/reorderable-table.component.less

```diff
@@ -2,6 +2,13 @@
     background-color: #fff;
 }
 
+.reorderable-row:focus {
```

Comment:
Can this be part of our stylesheet too? We already have styles for rows which can receive focus so we should be able to reuse those.